pool/test/tenant_test.ml

```diff
@@ -302,6 +303,27 @@ let update_smtp_auth () =
       events)
 ;;
 
+let delete_smtp_auth () =
+  let open Email in
+  let id = System_event.Id.create () in
+  let smtp_auth = Data.Smtp.(create () |> from_write) in
+  let smtp_id = SmtpAuth.(smtp_auth.id) in
```

Comment:
Pattern question/suggestion (no need to change):
We'd usually write `smtp_auth.SmtpAuth.id` which is the same, just without brackets.